app/javascript/react/screens/App/Plan/components/PlanRequestDetailList/PlanRequestDetailList.js

```diff
@@ -233,6 +235,41 @@ class PlanRequestDetailList extends React.Component {
     );
   };
 
+  onSelect = (eventKey, task) => {
+    const { downloadLogAction, fetchOrchestrationStackUrl, fetchOrchestrationStackAction } = this.props;
+    if (eventKey === 'migration') {
+      downloadLogAction(task);
+    } else {
+      fetchOrchestrationStackAction(fetchOrchestrationStackUrl, eventKey, task);
+    }
+  };
+
+  overlayTriggerClick = task => {
+    if (task.options.playbooks) {
+      const playbookStatuses = task.options.playbooks;
+      let runningPlaybook;
```

Comment:
another nit... any chance we can set this = null?  I know I know, undefined will also be evaluated to false on line 258.... 😬 (if not 'saul good 🤗)